chemprop/featurizers/molecule.py

```diff
@@ -9,7 +11,7 @@
 MoleculeFeaturizerRegistry = ClassRegistry[VectorFeaturizer[Mol]]()
 
 
-class MorganFeaturizerMixin:
+class MorganFeaturizerMixin(VectorFeaturizer[Mol]):
```

Comment:
The `MorganFeaturizerMixin` lacks a `__call__` function to convert input `Mol`s into output vectors. Therefore, I think the original version is ok.

Reply:
Good point